### PR TITLE
Buttons are added to Graph Hbox, not outside it

### PR DESCRIPTION
**Pull request by bits-by-brandon**
## Description

Edits button logic to insert Make Floating button and Toggle Panel button inside the GraphEdit HBox instead of beside it.
Allows buttons to be clicked

Note: This doesn't fix the icons for me, but at least fixes the clicking.

## Addressed issues

- https://github.com/bitbrain/beehave/issues/256

## Screenshots
Screenshot 2023-11-30 at 7 37 36 PM

### CODE REVIEW

**Comment by bits-by-brandon** [comment]
Register singleton 'regex_richtext:<RegEx#-9223371819002624369>'

What's this testing for? Is this flake or should I fix something?

File: addons/beehave/debug/debugger_tab.gd [hunk 1]
```diff
@@ -45,6 +46,7 @@ func _ready() -> void:
 
 	var toggle_button := Button.new()
 	toggle_button.flat = true
+	toggle_button.name = "TogglePanelButton"
```
Comment (bits-by-brandon) [inline addons/beehave/debug/debugger_tab.gd:49]: These names are just for helpful debugging in the Remote scene view

**Comment by bits-by-brandon** [comment]
Still getting this very slightly out of acceptable range error:

```
Run Test Suite res://test/randomized_composites/weighted_sampling/selector_random/selector_random_weights_test.gd 
	Run Test: res://test/randomized_composites/weighted_sampling/selector_random/selector_random_weights_test.gd > test_weights_effecting_sample :STARTED
Register singleton 'GdUnitDefaultValueDecoders:<RefCounted#-9223371819304614250>'
	Run Test: res://test/randomized_composites/weighted_sampling/selector_random/selector_random_weights_test.gd > test_weights_effecting_sample :FAILED 262ms
	Report:
Register singleton 'regex_richtext:<RegEx#-9223371819002624369>'
		line 36: Expecting:
		 '54.014599'
		 in range between
		 '46.000000' <> '54.000000'
```

**Comment by bitbrain** [comment]
Those are flaky tests in the main branch I aim to fix today. Sorry for that!

**Comment by bitbrain** [comment]
Screenshot 2023-12-03 102316

This is how this PR looks on my end. Good job! I will fix the icon scaling separately.

**Review by bitbrain — APPROVED** [review]

**Pull request merged**